src/__tests__/lib/command/util/apps-util-input-primitives.test.ts

```diff
@@ -20,27 +20,31 @@ jest.unstable_mockModule('../../../../lib/item-input/index.js', () => ({
 }))
 
 
-// This module has no functions. Test to make sure the variables call the correct functions.
-// To be honest, this is mostly just to get coverage.
-test('module', async () => {
-	const oauthAppScopeDefMock = buildInputDefMock<string[]>('Scopes Mock')
-	checkboxDefMock.mockReturnValueOnce(oauthAppScopeDefMock)
-
-	const redirectUriDefMock = buildInputDefMock<string>('Redirect URI Mock')
-	stringDefMock.mockReturnValueOnce(redirectUriDefMock)
-	const redirectUrisDefMock = buildInputDefMock<string[]>('Redirect URIs Mock')
-	arrayDefMock.mockReturnValueOnce(redirectUrisDefMock)
-
-	const {
-		oauthAppScopeDef,
-		redirectUrisDef,
-	} = await import('../../../../lib/command/util/apps-util-input-primitives.js')
-
-	expect(oauthAppScopeDef).toBe(oauthAppScopeDefMock)
-	expect(redirectUrisDef).toBe(redirectUrisDefMock)
-	expect(arrayDefMock).toHaveBeenCalledExactlyOnceWith(
-		'Redirect URIs',
-		redirectUriDefMock,
-		expect.objectContaining({ minItems: 0 }),
-	)
+const oauthAppScopeDefMock = buildInputDefMock<string[]>('Scopes Mock')
+checkboxDefMock.mockReturnValueOnce(oauthAppScopeDefMock)
+
+const redirectUriDefMock = buildInputDefMock<string>('Redirect URI Mock')
+stringDefMock.mockReturnValueOnce(redirectUriDefMock)
+const redirectUrisDefMock = buildInputDefMock<string[]>('Redirect URIs Mock')
+arrayDefMock.mockReturnValueOnce(redirectUrisDefMock)
+
+const {
+	oauthAppScopeDef,
```

Comment:
Similar to other comments, there's an alternate option here of `oAuthScopeDef`.  I think I like what you've got better than capitalizing the `A`, but wanted to point it out just in case.